### PR TITLE
facet: fix year

### PR DESCRIPTION
**Pull request by rerowep**
* Fixes year facet for imports.
* Better publication year facet for documents.

### CODE REVIEW

File: rero_ils/config.py [hunk 1]
```diff
@@ -2274,7 +2265,7 @@ def _(x):
             )
         ),
         filters={
-            _('request_status'): and_term_filter('status'),
+            _('status'): and_term_filter('status'),
```
Comment (PascalRepond) [inline rero_ils/config.py:2268]: This should not be modified. It was just fixed, the name of the facet should be 'request_status'

**Review by PascalRepond — CHANGES_REQUESTED** [review]

File: rero_ils/config.py [hunk 2]
```diff
@@ -1874,7 +1865,7 @@ def _(x):
             item_type=dict(terms=dict(field='item_type.pid', size=RERO_ILS_DEFAULT_AGGREGATION_SIZE)),
             temporary_location=dict(terms=dict(field='temporary_location.pid', size=RERO_ILS_DEFAULT_AGGREGATION_SIZE)),
             temporary_item_type=dict(terms=dict(field='temporary_item_type.pid', size=RERO_ILS_DEFAULT_AGGREGATION_SIZE)),
-            status=dict(terms=dict(field='status', size=RERO_ILS_DEFAULT_AGGREGATION_SIZE)),
+            request_status=dict(terms=dict(field='status', size=RERO_ILS_DEFAULT_AGGREGATION_SIZE)),
```
Comment (PascalRepond) [inline rero_ils/config.py:1868]: Why?

File: rero_ils/config.py (hunk 1 above)
Comment (PascalRepond) [reply rero_ils/config.py:2269]: Not resolved.

File: rero_ils/config.py [hunk 3]
```diff
@@ -1888,7 +1879,7 @@ def _(x):
             _('item_type'): and_term_filter('item_type.pid'),
             _('temporary_item_type'): and_term_filter('temporary_item_type.pid'),
             _('temporary_location'): and_term_filter('temporary_location.pid'),
-            _('status'): and_term_filter('status'),
+            _('request_status'): and_term_filter('status'),
```
Comment (PascalRepond) [inline rero_ils/config.py:1882]: Why?

**Review by PascalRepond — CHANGES_REQUESTED** [review]

**Review by lauren-d — APPROVED** [review]

**Pull request merged**